cherry_picker/test_cherry_picker.py

```diff
@@ -110,7 +111,16 @@ def git_config():
 def tmp_git_repo_dir(tmpdir, cd, git_init, git_commit, git_config):
     repo_dir = tmpdir.mkdir("tmp-git-repo")
     cd(repo_dir)
-    git_init()
+    try:
+        git_init()
+    except subprocess.CalledProcessError:
+        version = subprocess.run(("git", "--version"), capture_output=True)
+        split_version = version.stdout.decode("utf--8").split(".")
+        if int(split_version[0][-1]) < 1 or int(split_version[1]) < 28:
+            warnings.warn(
+                "You need git 2.28.0 or newer to run the full test suite.",
+                UserWarning,
+            )
```

Comment:
IIUC this will continue to run and result in a number of failures, right?
Is the warning easily visible among the failures?  If not, a skip (or even a `raise`) might be better, as long as it's evident why the tests can't be run.

Reply:
<img width="885" alt="Schermafbeelding 2022-07-15 om 09 06 02" src="https://user-images.githubusercontent.com/13665637/179170421-78008da0-c1dc-403b-abb6-fdb523e55920.png">

I think this is pretty clear, but let me know if you'd like to change it.